layouts/joomla/mail/mailtemplate.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_mails
```

Comment:
```suggestion
 * @subpackage  Layout
```

that's how all the other layouts are documented